tests/test_service.py

```diff
@@ -107,30 +116,7 @@ class Bar(metaclass=service.Meta):
         class Baz(metaclass=service.Meta):
             ORDER_BEFORE = [Bar]
 
-        self.assertSetEqual(
-            {Foo},
-            Bar.ORDER_BEFORE
-        )
-        self.assertSetEqual(
-            {Foo, Bar},
-            Baz.ORDER_BEFORE
-        )
-        self.assertSetEqual(
-            {Bar, Baz},
-            Foo.ORDER_AFTER
-        )
-        self.assertSetEqual(
-            {Baz},
-            Bar.ORDER_AFTER
-        )
-        self.assertSetEqual(
-            set(),
-            Foo.ORDER_BEFORE
-        )
-        self.assertSetEqual(
-            set(),
-            Baz.ORDER_AFTER
-        )
+        self.assertTrue(Foo >= Bar >= Baz)
```

Comment:
I prefer to avoid ``assertTrue`` whenever possible (e.g. there is no ``assertIsSubclass(…)``, so I have to use ``assertTrue(issubclass(…))``), because the error messages of the specific methods are more explicit.

If you don’t have a strong rationale here for doing otherwise, please use ``assertGreaterEqual`` instead.